tests/components/selectBox/index.test.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import SelectBox from '../../../app/js/components/selectBox';
+
+let props;
+let mountedComponent;
+
+const getComponent = () => {
+  if (!mountedComponent) {
+    mountedComponent = shallow(<SelectBox {...props} />);
+  }
+  return mountedComponent;
+};
+
+describe('Component: SelectBox', () => {
+  beforeEach(() => {
+    props = {
+      name: 'sets',
+      items: [{ value: 'Choose a set to add', id: 0 }, { value: 'Mineral', id: 1 }],
+      selectedItem: {},
+      onChange: jest.fn(),
+    };
+    mountedComponent = undefined;
+  });
+
+  it('renders properly', () => {
+    const component = getComponent();
+    expect(component).toMatchSnapshot();
+  });
+
+  it('should call dropDown()', () => {
```

Comment:
please refactor the message based on comment below

Reply:
Refactored